spec/features/admin/show_users_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe "admin" do
+  describe "can see all users through" do
+    it "the all users page" do
+      admin = User.create!(name: "bob", password: '12345', password_confirmation: '12345', address: "street", city: "Denver", state: "CO", zip:"12345", email: "someone@gmail.com", role: 1)
+      user1 = User.create!(name: "jerry", password: '12345', password_confirmation: '12345', address: "street", city: "Denver", state: "CO", zip:"12345", email: "jerry@gmail.com", role: 0)
+      user2 = User.create!(name: "nancy", password: '12345', password_confirmation: '12345', address: "street", city: "Denver", state: "CO", zip:"12345", email: "nancy@gmail.com", role: 0)
+      user3 = User.create!(name: "john", password: '12345', password_confirmation: '12345', address: "street", city: "Denver", state: "CO", zip:"12345", email: "john@gmail.com", role: 0)
+      
```

Comment:
Now I will search for my comment about how I thought seeing all the users was a problem.  I understand the detail I missed.